default/stringtables/en.txt

```diff
@@ -3897,6 +3897,14 @@ ENC_SUITABILITY_REPORT_WHEEL_INTRO
 Suitability is determined by distance from the preferred type of [[encyclopedia ENVIRONMENT_TITLE]] in this wheel:
 '''
 
+ENC_EMPIRE_DATA
+[Current Game - Universe]
```

Comment:
Changed to `ENC_EMPIRE_UNIVERSE_DATA` to reflect a limited view of the universe.